plugins.star

```diff
@@ -19,6 +19,8 @@ plugins_map = {
     "AutoGPTUserInput": {"repository": "HFrovinJensen/Auto-GPT-User-Input-Plugin", "branch": MASTER_BRANCH},
     "BingAI": {"repository": "gravelBridge/AutoGPT-BingAI", "branch": MASTER_BRANCH},
     "AutoGPTCryptoPlugin": {"repository": "isaiahbjork/Auto-GPT-Crypto-Plugin", "branch": MASTER_BRANCH},
+    "AutoGPT_YouTube": {"repository": "jpetzke/AutoGPT-YouTube", "branch": MASTER_BRANCH},
+    "AutoGPT-Zapier": {"repository": "AutoGPT-Zapier", "branch": MASTER_BRANCH},
```

Comment:
the author of the repo is missing! please put the author/reponame

Reply:
already fixed it; forgot to commit that one in previous one. @h4ck3rk3y